bedrock/base/templates/includes/site-footer.html

```diff
@@ -58,7 +58,12 @@
     <nav class="secondary">
         <div class="small-links">
           <ul>
-            <li><a rel="nofollow" href="{{ url('privacy') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
+            {% if LANG == 'en-US' %} 
```

Comment:
Appologies for missing this the first time around, but you could also expand coverage of the new strings here by using `{% if LANG.startswith('en-') %}`, as opposed to just specifying `en-US`. This also applies to the other occurances in this PR.